tycho-versions-plugin/src/main/java/org/eclipse/tycho/versions/pom/PomFile.java

```diff
@@ -245,4 +276,25 @@ private String getElementValue(String name) {
     public boolean isMutable() {
         return isMutable;
     }
+
+    private static boolean isCiFriendly(String v) {
```

Comment:
can we make this a public static  somewhere as it seems to be used in more than one place now in the `PomFile` and `PomManipulator`?

Reply:
should this be deleted now?